frontend/src/container/withActivity.jsx

```diff
@@ -268,45 +270,72 @@ const withActivity = (WrappedComponent, setActivityList, setActivityNextPage, re
 
     isLoggedUserMember = (activity) => this.props.workspaceList.find(space => space.id === activity.newestMessage.fields.workspace.workspace_id)
 
-    isNotPublicationOrInWorkspaceWithActivatedPublications = (activity) => {
-      const { props } = this
-      if (activity.content.content_namespace !== CONTENT_NAMESPACE.PUBLICATION ||
-        !activity.newestMessage.fields.workspace) return true
-      const currentWorkspace = props.workspaceList.find(ws => ws.id === activity.newestMessage.fields.workspace.workspace_id)
-      if (!currentWorkspace) return true
-      return currentWorkspace.publicationEnabled
+    isActuality = (activity) => {
+      const isActuality = activity.content?.content_namespace === CONTENT_NAMESPACE.PUBLICATION
+      const hasSpace = activity.newestMessage.fields.workspace
+      return isActuality || hasSpace
```

Comment:
If I understand well you transformed 
```
(activity.content.content_namespace !== CONTENT_NAMESPACE.PUBLICATION || !activity.newestMessage.fields.workspace)
```
to
```
!(isActuality || hasSpace)
```
If yes, that is not the same thing.

A | B | ∼ A ∨ ∼ B
-- | -- | --
1 | 1 | 0
1 | 0 | 1
0 | 1 | 1
0 | 0 | 1


A | B | ∼ ( A ∨ B )
-- | -- | --
1 | 1 | 0
1 | 0 | 0
0 | 1 | 0
0 | 0 | 1

You should return isActuality && hasSpace here.



A | B | ∼ ( A ∧ B )
-- | -- | --
1 | 1 | 0
1 | 0 | 1
0 | 1 | 1
0 | 0 | 1